reports/semgrep_report.json

```diff
@@ -0,0 +1,17 @@
+veracode/check_compliance.py
+severity:error rule:python.lang.security.use-defused-xml.use-defused-xml: Found use of the native Python XML libraries, which is vulnerable to XML external entity (XXE)
+attacks. The Python documentation recommends the 'defusedxml' library instead if the XML being
+loaded is untrusted.
+
+9:from xml.etree import (  # nosec (Used only when TYPE_CHECKING)
```

Comment:
@pb-seiso Semgrep maintainer here 👋  

Did you see [Semgrep Docs > Managing Findings](https://semgrep.dev/docs/running-rules/#managing-findings) for details on Semgrep's `nosec`-like feature? I think it'll do what you want for suppression, and happy to open tickets on Semgrep if not!

Reply:
Thanks Luke! I missed that when reviewing the docs. That's exactly what I was looking for.